verilog/formatting/formatter_test.cc

```diff
@@ -2589,23 +2582,18 @@ static const std::initializer_list<FormatterTestCase>
          "        end else if (oo) begin\n"
          "          aa = bb;\n"
          "          cc = dd;\n"
-         "          if (zz)\n"
-         "            zx = xz;\n"
-         "          else\n"
-         "            ba = ab;\n"
+         "          if (zz) zx = xz;\n"
+         "          else ba = ab;\n"
          "        end else begin\n"
          "          vv = tt;\n"
          "        end\n"
          "      end\n"
          "    end else begin\n"
-         "      if (x)\n"
-         "        y = a;\n"
+         "      if (x) y = a;\n"
          "      else begin\n"
          "        a = y;\n"
-         "        if (a)\n"
-         "          b = c;\n"
-         "        else\n"
-         "          d = e;\n"
+         "        if (a) b = c;\n"
+         "        else d = e;\n"
          "      end\n"
          "    end\n"
          "  end\n"
```

Comment:
one more suggested test case:

```
if () ...
else if () ...
else ...
```

(uses both else-if and final-else)

Reply:
Done